CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 3.0.0 (to be released)
+
+* [BC] removed deprecated features
+* [BC] made final all classes that implement an interface
+* Dropped support for KnpMenu 2
+* Increased minimum PHP version to 7.2 (to be consistent with KnpMenu 3)
```

Comment:
The fact that class parameters are removed in the container should also be documented.